template-parts/updatebox-section.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Climate Hub partners page supplementary updatebox section template
+ * @package climatehub
+ */
+
+
+  if(get_field('enable_custom_field')): ?>
```

Comment:
Can you fix the indenting in this file to make sure that it's all even. The `if` doesn't need to be indented on this line. Can you also rename the custom field to something more specific than `enable_custom_field` this will make sure we don't have any conflicts.